advanced-usage/package.json

```diff
@@ -8,6 +8,9 @@
     "start": "parcel index.html --open",
     "build": "parcel build index.html"
   },
+  "dependencies": {
+    "hyperformula": "0.1.0"
```

Comment:
If, we're going to support whole `0.1.x` family of releases with those examples shouldn't we have some kind of wildcard here?

Reply:
True!